pkg/remote/deploy.go

```diff
@@ -160,8 +158,8 @@ func DeployRemote(remoteDeployOptions *DeployOptions) (*DeploymentResult, *RemIn
 		Namespace:          namespace,
 		WorkspaceID:        workspaceID,
 		PVCName:            workspacePVC,
-		ServiceAccountName: serviceAccountName,
-		PullSecret:         secretName,
+		ServiceAccountName: "codewind-" + workspaceID, //  codewind-k39vwfk0
```

Comment:
Did you mean for this comment to be here?

Reply:
Yeah because I wanted to remind myself what format this service account name was in rather than re append the Workspace ID a second time downstream by mistake